inclusive-design/wecount.inclusivedesign.ca
# ci: only run lint GitHub action when relevent files have been modified

* [x ] This pull request has been tested by running `npm run start` and reviewing affected routes

## Description

This PR configures the `paths` property for the linting GitHub action to only trigger the action when JavaScript or Sass files have been changed so as to reduce unnecessary action runs.

## Steps to test

1. Open a pull request or commit changes to any branch.

**Expected behavior:** The lint action only runs when JavaScript and Sass files have been changed.

## Additional information

Not applicable.

## Related issues

Not applicable.
